easybuild/tools/modules.py

```diff
@@ -487,7 +487,10 @@ def module_wrapper_exists(self, mod_name, modulerc_fn='.modulerc', mod_wrapper_r
         """
         Determine whether a module wrapper with specified name exists.
         Only .modulerc file in Tcl syntax is considered here.
+        DEPRECATED. Use exists()
         """
+        self.log.deprecated('module_wrapper_exists is unreliable and should no longer be used', '5.0')
```

Comment:
Maybe also mention that the `exists` method should be used?

```python
self.log.deprecated('module_wrapper_exists is unreliable, use exists method instead', '5.0')

Reply:
Done a bit more verbose dace69d1d